src/dxtbx/boost_python/flumpy.cc

```diff
@@ -586,8 +636,11 @@ PYBIND11_MODULE(dxtbx_flumpy, m) {
         &vecs_from_numpy,
         "Convert a numpy object to a flex.vec2 or .vec3, depending on input array");
   m.def("mat3_from_numpy", &mat3_from_numpy, "Convert a numpy object to a flex.mat3");
+  m.def("miller_index_from_numpy",
+        &miller_index_from_numpy,
+        "Convert a numpy object to a flex.miller_index");
 
   // Make sure that we have imported flex - cannot do boost::python conversions
   // otherwise
-  pybind11::module::import("scitbx.array_family.flex");
+  pybind11::module::import("cctbx.array_family.flex");
```

Comment:
```suggestion
  pybind11::module::import("scitbx.array_family.flex");
  pybind11::module::import("cctbx.array_family.flex");
```
We want to still be explicit here, rather than rely on implicit `cctbx` importing behaviour.